src/snmalloc/mem/localalloc.h

```diff
@@ -186,6 +187,10 @@ namespace snmalloc
         return small_alloc<NoZero>(1);
       }
 
+      void* result = SecondaryAllocator::allocate(size);
+      if (result != nullptr)
+        return capptr::Alloc<void>::unsafe_from(result);
+
```

Comment:
I think we should probably move this inside the `check_init` call.  I.e. just before `// Grab slab of correct size`, I think we might have initialisation issues if we haven't fully initialised snmalloc as the pagemap won't exist when we look up a GWP Asan allocation.